db/migrate/20170728004206_update_terms_and_conditions_yes_content.rb

```diff
@@ -0,0 +1,15 @@
+class UpdateTermsAndConditionsYesContent < ActiveRecord::Migration
```

Comment:
It would be helpful to say that this is updating the mhvac terms and conditions.

The terms and conditions table could technically have rows for any service, this one is specifically modifying version 1.0 of 'mhvac'